### PR TITLE
docs: README refactor

### PR DESCRIPTION
**Pull request by teolemon**
### What

- [x] README refactor
- [x] Added the Weekly Call
- [x] Added the Sprint board
- [x] Collapsed some of the information to make the README more readable

### CODE REVIEW

**Review by alexgarel — APPROVED** [review]
Good move !

File: README.md [hunk 1]
```diff
@@ -37,9 +42,27 @@ Open Food Facts is a non-profit association of volunteers.
...
 
 Data about food is of public interest and has to be open (i.e available to everyone). The complete database is published as open data and can be reused by anyone and for any use. Check-out the cool reuses or make your own!
+</details>
 
 * <https://world.openfoodfacts.org>
 
+## Feature Sprint 
+- We use feature-based sprints, [tracked here](https://github.com/orgs/openfoodfacts/projects/32)
+
+<details><summary><h2> Weekly meetings </h2></summary>
```
Comment (stephanegigandet) [inline README.md:52]: This part does not render correctly.

image

Comment (teolemon) [reply README.md:52]: Je vais enlever le collapse voir si ça règle le souci

Comment (teolemon) [reply README.md]: works now, was a missing carriage return

**Pull request merged**